code/game/objects/structures/aliens.dm

```diff
@@ -126,6 +126,8 @@
 	canSmoothWith = list(/obj/structure/alien/weeds, /turf/closed/wall)
 	smooth = SMOOTH_MORE
 
+var/list/blacklisted_weed_turfs = typecacheof(list(/turf/open/floor/plating/lava))
```

Comment:
Oh, to add to the confusion, point 1 above is incorrect, because it's a ```/static``` it's compiled as a global, so it's both in the object definition and not, that's why it's compiling despite bgo putting it in the def and not ```New()```, a few places have done this too actually (like Clockcult)